index.html

```diff
@@ -9,52 +9,67 @@
-                        <section class="intro">
-                            <h1>I’m Karthika. <br>
-                                Glad to see <br> you!</h1>
-                            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
-                                 If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                        </section>
-                        <section>
-                            <div class="connect">
-                                <h4>LET'S CONNECT</h4>
-                            </div>
-                            
-                            <div class="social">
-                                <ul>
-                                    <li><a href="https://twitter.com/PorkalaiKarthi"><img src="Vector.png" alt="Twitter"></a> </li>
-                                    <li><a href="https://www.linkedin.com/in/porkalai-karthika/"> <img src="Linkedin icon.png" alt="Linked in"></a></li>
-                                    <li><a href="https://medium.com/"><img src="msg.png" alt="message"></a></li>
-                                    <li><a href="https://github.com/Porkalaikarthika"> <img src="github.png" alt="github"></a></li>
-                                    <li><a href="https://www.angellist.com/">  <img src="finger.png" alt="finger"> </a></li>
-                                </ul>  
-                            </div>
-                        </section>
+        <section>
+            <div class="container">
+                <header>
+                 <nav class="menu">
+                     <a href="#">Porkalai</a>                        
+                     <img src="Icon - Menu.png" alt="Icon-menu">
+                 </nav>
+                 <nav class="menuBar">
+                    <a href="#">Porkalai</a>
+                    <div class="menuFlex">
+                        <p>Portfolio</p>
+                        <p>About</p>
+                        <p>Contact</p>
+                    </div>
+                  </nav>                    
+                </header>
+            <div class="outerbg">
+            <div class="content">
+                <div class="intro">
+                    <h1 class="mimg">I’m Karthika. <br>
+                        Glad to see <br> you!</h1>
+                        <h1 class="dimg">
+                            I’m Karthika. <br>
+                        Glad to see you!
+                        </h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+                         If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                    </div>
+                <div>
+                    <div class="connect">
+                        <h4>LET'S CONNECT</h4>
                     </div>
```

Comment:
- [ ] As stated by the previous reviewer 😃 kindly try to fix the responsive as this is causing your design to only look good on screen widths of 375px and 1440px. Please note that you are supposed to use responsive values for your element sizes so that the webpage scales properly whenever the screen size is increased or reduced. Take a look at the video below for reference:

- Click at this [link here](https://www.loom.com/share/dbc0181598044568ac9975bb0fd50a33) to see the video demo for your design 😊⤵️